executor/write.go

```diff
@@ -1110,7 +1110,8 @@ type UpdateExec struct {
 	OrderedList []*expression.Assignment
 
 	// updatedRowKeys is a map for unique (Table, handle) pair.
-	updatedRowKeys map[table.Table]map[int64]struct{}
+	// According to issue#3604, TableName is better than table.Table.
```

Comment:
Of course. Interface is not good choice as map key.
I just wonder that is the table name case-insensitive?